### ISSUE TITLE
Setup guest wifi in office

### ISSUE THREAD

**Issue by KyleKrukar** [bythepixel/devhour#13]
wifi-name: devhour
pw: hackerman

?

**Comment by KyleKrukar** [comment]
@wreyniers can you whip this one up by tomorrow?

**Comment by rreynier** [comment]
Turned it on. Username is bpt-guest for re-useability. Password is hackerman is requested.

**Comment by KyleKrukar** [comment]
@rreynier I don't see the guest network. Can you look into this today pls?

**Comment by rreynier** [comment]
Because this is becoming a small can of worms, let's just use our regular wifi. Not really sure what a guest wifi would really offer anyway.

**Comment by KyleKrukar** [comment]
@rreynier 

Guest networks are used to keep things separate and secure... like.. to keep someone from cranking Brittney Spears on our stereo. Or to keep someone from printing 2,000 copies of Nick Cage on our printer from the sidewalk.

It's best to set up a guest network for these things IMO.

**Comment by KyleKrukar** [comment]
Oh yeah, and totally fine for using ours for this one.

**Comment by rreynier** [comment]
You are right. Especially since we plan on growing this. Wouter can update the router firmware during off-hours and that should get us going.  Ill assign the task to him!
